x-pack/plugins/index_management/__jest__/client_integration/home/indices_tab.test.ts

```diff
@@ -161,4 +221,48 @@ describe('<IndexManagementHome />', () => {
       expect(latestRequest.url).toBe(`${API_BASE_PATH}/settings/${encodeURIComponent(indexName)}`);
     });
   });
+
+  describe('index detail panel actions', () => {
+    const indexName = 'testIndex';
+    beforeEach(async () => {
+      const index = {
+        health: 'green',
+        status: 'open',
+        primary: 1,
+        replica: 1,
+        documents: 10000,
+        documents_deleted: 100,
+        size: '156kb',
+        primary_size: '156kb',
+        name: indexName,
+      };
+      httpRequestsMockHelpers.setLoadIndicesResponse([index]);
+
+      testBed = await setup();
+      const { component, find } = testBed;
+
+      component.update();
+
+      find('indexTableIndexNameLink').at(0).simulate('click');
+    });
+
+    test('should be able to close index', async () => {
+      const { find, actions } = testBed;
+
+      actions.clickManageContextMenuButton();
+      const contextMenu = find('indexContextMenu');
+
+      const closeIndexButton = contextMenu
+        .childAt(0)
+        .childAt(0)
+        .childAt(1)
+        .childAt(0)
+        .find('button[data-test-subj="indexTableContextMenuButton"]')
+        .at(0);
```

Comment:
This code looks a bit confusing, I think that's because the context buttons don't have specific `data-test-subj` values. I suggest add a value in this [file](https://github.com/elastic/kibana/blob/master/x-pack/plugins/index_management/public/application/sections/home/index_list/index_actions_context_menu/index_actions_context_menu.js) to use in the tests.